tests/test_prepdocslib_jsontextsplitter.py

```diff
@@ -0,0 +1,34 @@
+from scripts.prepdocslib.page import Page
```

Comment:
FYI, for the tests, you can run pytest --cov to see coverage stats for your new files